### PR TITLE
Remove the 'Coalesced' grammar flag.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Fixes #44

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -24,20 +24,17 @@ <h1>Binary Logical Operators</h1>
...
-      <ins>[~Coalesced] LogicalORExpression[?In, ?Yield, ?Await, ~Coalesced] `||` LogicalANDExpression[?In, ?Yield, ?Await, ~Coalesced]</ins>
+    CoalesceExpression[In, Yield, Await] :
+      CoalesceExpressionHead[?In, ?Yield, ?Await] `??` BitwiseORExpression[?In, ?Yield, ?Await]
 
-    LogicalANDExpression[In, Yield, Await, <ins>Coalesced</ins>] :
-      BitwiseORExpression[?In, ?Yield, ?Await]
-      <del>LogicalANDExpression[?In, ?Yield, ?Await] `&amp;&amp;` BitwiseORExpression[?In, ?Yield, ?Await]</del>
-      <ins>[~Coalesced] LogicalANDExpression[?In, ?Yield, ?Await, ~Coalesced] `&amp;&amp;` BitwiseORExpression[?In, ?Yield, ?Await]</ins>
+    CoalesceExpressionHead[In, Yield, Await] :
+      CoalesceExpression
```
Comment (waldemarhorwat) [inline spec.html]: Why omit the grammar parameters here?

File: spec.html [hunk 2]
```diff
@@ -24,20 +24,17 @@ <h1>Binary Logical Operators</h1>
   <h2>Syntax</h2>
   <emu-grammar type="definition">
     <ins>
-    CoalesceExpression[In, Yield, Await, Coalesced] :
-      LogicalORExpression[?In, ?Yield, ?Await, ?Coalesced]
-      CoalesceExpression[?In, ?Yield, ?Await, +Coalesced] `??` LogicalORExpression[?In, ?Yield, ?Await, +Coalesced]
-    </ins>
+    LogicalExpression[In, Yield, Await] :
```
Comment (jridgewell) [inline spec.html]: I think we need a change to ternary's grammar, too.

Comment (DanielRosenwasser) [reply spec.html]: That's lower down (https://github.com/tc39/proposal-nullish-coalescing/pull/45/files/a4f09e6d84b9fda817b1e8ad77e4d33069515652#diff-3540caefa502006d8a33cb1385720803R96), but the diff doesn't show enough context. Is that what you meant?

Comment (jridgewell) [reply spec.html]: Missed that, I only saw the `IsFunctionDefinition` and below.

**Pull request merged**